src/models/resources.ts

```diff
@@ -0,0 +1,206 @@
+import {
+	PineResourceBase,
+	NavigationResource,
+	ReverseNavigationResource,
+} from './utils';
+
+export interface User extends PineResourceBase {
```

Comment:
A great question here is whether we would prefer splitting all these interfaces in separate files.
Having a single huge one might arguably make it slower to find what you are searching.
I have the same dilemma in the SDK, but I'm grabbing the opportunity to discuss this in here.
PS: I'm in favor of slitting into separate files.